tests/sentinel/run.tcl

```diff
@@ -17,11 +17,13 @@ proc main {} {
         "sentinel deny-scripts-reconfig no"
         "enable-protected-configs yes"
         "enable-debug-command yes"
+        "save ''"
```

Comment:
i guess in sentinel instance, we can leave it alone. (but i choose to add it)

Reply:
it doesn't really matter, but i'd say that it should be an invalid config in sentinel, so i'd rather drop it.